packages/synapse-react-client/esbuild.config.mjs

```diff
@@ -15,7 +15,6 @@ const globals = {
   'rss-parser': 'Parser',
   'react-mailchimp-subscribe': 'ReactMailchimpSubscribe',
   'plotly.js-basic-dist': 'Plotly',
-  '@rjsf/core': 'JSONSchemaForm',
```

Comment:
Let's include RJSF in the UMD bundle. Our fork was included before removing it in this PR, so bundle size should not change much.